src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -8,16 +8,15 @@ import { scrollAndFocus } from '../../common/utils/helpers';
 import PhotoPreview from '../components/PhotoPreview';
 import CropperController from '../components/CropperController';
 
-const LARGE_SCREEN = 1201;
 const MIN_SIZE = 350;
 
-function isSmallScreen(width) {
-  return  width < LARGE_SCREEN;
-}
-
-function onReviewPage(pageTitle) {
```

Comment:
This was used to properly style the cropper on the review page, we will now want to make sure we detect the parent component's width (potentially via an initial empty render used to capture this information), rather than the window width.

Reply:
I understand now. I will have to pass a prop into the component to stop the `detectWidth` from firing and use that to control the markup if it's passed in.